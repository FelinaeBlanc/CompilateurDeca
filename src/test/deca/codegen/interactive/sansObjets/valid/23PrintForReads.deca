{

    print(readInt());
    println();
    println(readInt()); 


    print(readFloat()); 
    println();
    println(readFloat()); 

    printx(readFloat()); 
    println();
    printlnx(readFloat()); 
}