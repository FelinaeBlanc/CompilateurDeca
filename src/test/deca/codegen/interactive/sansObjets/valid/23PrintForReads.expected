125
10
1.52580e+01
1.24870e+01
0x1.16774ap+12
0x1.61b08ap-2
